_pages/platform/SAML-authentication-org-management.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: article
+title:  "SAML Authentication and Organization Management"
+excerpt: ""
+categories: basics
+nav_index: 40
+show_related: true
+image:
+teaser: /icons/algo.svg
+permalink: /platform/saml-authentication/
+redirect_from:
+-  /basics/saml-authentication/
+---
+
+Security Assertion Markup Language (SAML) is an open standard that allows an identity provider (IdP) to pass authorization credentials to a service provider (SP).
+
+SAML authentication allows a user to use SAML assertions in order to log in to the platform. If configured, permission tags can be extracted from the SAML assertion and can be used to facilitate external management of organization membership and user security roles.
+
+This feature is available to [Algorithmia Enterprise](/enterprise) users who have configured SAML and external organization management functionality only.
+{: .notice-enterprise}
+
+#### Use cases
+-  Allowing user login through SAML
+-  Using an external management platform for user permission (admin/sudo)
+-  Using an external management platform for granting/revoking organization membership/administration privileges
+-  Creating local users that represent validated users verified through SAML
```

Comment:
technical term would be authenticated through SAML, just figured id mention that incase it mattered, not sure if a change is needed

Reply:
will make this change, thanks